ibm/resource_ibm_cbr_zone.go

```diff
@@ -0,0 +1,682 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/contextbasedrestrictionsv1"
+)
+
+func resourceIBMCbrZone() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMCbrZoneCreate,
+		ReadContext:   resourceIBMCbrZoneRead,
+		UpdateContext: resourceIBMCbrZoneUpdate,
+		DeleteContext: resourceIBMCbrZoneDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: InvokeValidator("ibm_cbr_zone", "name"),
+				Description:  "The name of the zone.",
+			},
+			"account_id": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
```

Comment:
make this as Computed field..
Instead of taking acountId  from the user.. we generally derive it internally..

Reply:
done